### DISCUSSION TITLE
Performance of attached process

### DISCUSSION THREAD

**Discussion by phamdu98** [alibaba/arthas#2607] [Q&A]
Hi everyone,
Fisrt of all, thank for this project to help me resolve a lot of problem in my work. But i have on question, if i attach the Arthas to my process all the time, will it affect on my process (slow or consume ram,cpu)?. Because some issue that happen at some time when i am not ready for it, so i want attach Arthas all the time to monitor as quicly as possible my process when some accident happen. 
Thank you

**Comment by hengyunabc** [comment]
https://arthas.aliyun.com/en/doc/faq.html
